### PR TITLE
HDDS-2659. KeyValueHandler#handleCreateContainer should log the exception on container creation failure

### PR DESCRIPTION
**Pull request by cxorm**
## What changes were proposed in this pull request?
Let the ```StorageContainerException``` cause of ```KeyValueHandler#handleCreateContaine``` be logged.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-2659

## How was this patch tested?
Just build and compile.

### CODE REVIEW

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java [hunk 1]
```diff
@@ -146,6 +146,7 @@ public static ContainerCommandResponseProto logAndReturnError(
     log.info("Operation: {} , Trace ID: {} , Message: {} , Result: {}",
         request.getCmdType().name(), request.getTraceID(),
         ex.getMessage(), ex.getResult().getValueDescriptor().getName());
+    log.error("StorageContainerException Occurred!", ex);
```
Comment (xiaoyuyao) [inline hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java]: We already have a log.info just above it with exception message and the result. If more information about the exception is needed, can we consolidate it into the previous message.

Comment (cxorm) [reply hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java]: Thanks @xiaoyuyao for the comment.
Updated.

**Comment by anuengineer** [comment]
Not sure why the checks failed. This patch should not cause it. I have requested a re-run of the tests.

**Comment by cxorm** [comment]
Thanks @xiaoyuyao, @anuengineer and @bharatviswa504 for the reviews.
And thanks @mukul1987 for the merge.

**Pull request merged**